src/Router.js

```diff
@@ -44,7 +44,7 @@ export default class Router extends React.Component {
   componentDidMount() {
     if (!this.props.dispatch) {
       navigationStore.dispatch = this.navigator._navigation.dispatch;
```

Comment:
Could we avoid using private properties like `_navigation` ?

Reply:
Ah I missed this one. I will see if I can get this one out of here too.